Description: Remove .pyc files and such from SOURCES.txt
 SOURCES.txt from upstream contains a bunch of artefacts, which we remove.
Author: Soren Hansen <soren@linux2go.dk>

--- python-cloudservers-1.0~a5.orig/python_cloudservers.egg-info/SOURCES.txt
+++ python-cloudservers-1.0~a5/python_cloudservers.egg-info/SOURCES.txt
@@ -31,29 +31,14 @@ python_cloudservers.egg-info/entry_point
 python_cloudservers.egg-info/requires.txt
 python_cloudservers.egg-info/top_level.txt
 tests/__init__.py
-tests/__init__.pyc
 tests/fakeserver.py
-tests/fakeserver.pyc
-tests/livetests.pyc
 tests/test_auth.py
-tests/test_auth.pyc
 tests/test_backup_schedules.py
-tests/test_backup_schedules.pyc
 tests/test_base.py
-tests/test_base.pyc
 tests/test_client.py
-tests/test_client.pyc
 tests/test_flavors.py
-tests/test_flavors.pyc
 tests/test_images.py
-tests/test_images.pyc
 tests/test_ipgroups.py
-tests/test_ipgroups.pyc
-tests/test_live.pyc
-tests/test_server_addresses.pyc
 tests/test_servers.py
-tests/test_servers.pyc
 tests/test_shell.py
-tests/test_shell.pyc
-tests/utils.py
-tests/utils.pyc
\ No newline at end of file
+tests/utils.py
\ No newline at end of file
